Resources/Prototypes/Catalog/Fills/Crates/food.yml

```diff
@@ -90,4 +103,43 @@
     - id: DrinkSodaWaterCan
       amount: 2
     - id: DrinkCreamCarton
-      amount: 2
\ No newline at end of file
+      amount: 2
+
+- type: entity
+  id: CrateFoodSoftdrinks
+  parent: CratePlastic
+  components:
+    - type: StorageFill
+      contents:
+      - id: DrinkColaCan
+        amount: 4
+      - id: DrinkGrapeCan
+        amount: 2
+      - id: DrinkRootBeerCan
+        amount: 2
+      - id: DrinkIcedTeaCan
+        amount: 2
+      - id: DrinkLemonLimeCan
+        amount: 2
+      - id: DrinkFourteenLokoCan
+        amount: 2
+
+- type: entity
+  id: CrateFoodSoftdrinksLarge
+  parent: CratePlastic
+  components:
+    - type: StorageFill
+      contents:
+        - id: DrinkColaCan
+          amount: 8
+        - id: DrinkGrapeCan
+          amount: 4
+        - id: DrinkRootBeerCan
+          amount: 4
+        - id: DrinkIcedTeaCan
+          amount: 4
+        - id: DrinkLemonLimeCan
+          amount: 4
+        - id: DrinkFourteenLokoCan
+          amount: 4
```

Comment:
These + the food crates are excessive, the food in particular invalidates the chef for a good 20-30 minutes.